DESCRIPTION

```diff
@@ -28,7 +28,7 @@ biocViews:
 License: MIT + file LICENSE
 URL: https://github.com/gagneurlab/FRASER
 BugRepots: https://github.com/gagneurlab/FRASER/issues
-RoxygenNote: 7.1.1
+RoxygenNote: 7.1.2
```

Comment:
we need to add Karoline too if we include this code here.

Reply:
true, I had added her as an author for the resultsAnnotation.Rd file, but it seems that that got lost when I made other changes there. But definitely makes sense to add her here as well